steps/ftp-upload/2.1.0/step.yml

```diff
@@ -0,0 +1,75 @@
+title: FTP Upload
+summary: You get what you see, upload a file or directory to the target directory
+description: Upload a single file or a folder (recursively) to an FTP server.
+website: https://github.com/bitrise-io/steps-ftp-upload
+source_code_url: https://github.com/bitrise-io/steps-ftp-upload
+support_url: https://github.com/bitrise-io/steps-ftp-upload/issues
+published_at: 2017-08-09T15:03:15.604286437+02:00
+source:
+  git: https://github.com/bitrise-io/steps-ftp-upload.git
+  commit: c9f3a830643a89d18c7796bc007a909ec25742b2
+host_os_tags:
+- osx-10.9
+type_tags:
+- ftp
```

Comment:
please update the `type_tags` based on: [step-development-guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention)